### PR TITLE
[ECM-713] Move Accepted Case to Case Closed state error check

### PR DESCRIPTION
**Pull request by phani-perla1**
### Change description ###

1. Move Accepted Case to Case Closed state error check
2. Move Rejected Case to Case Closed state 
3. Move Submitted Case to Case Closed state error check
4. Code Improvements

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/test/end-to-end/paths/moveAcceptedCaseToCaseCloseStateErrorCheck.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const testConfig = require('./../../config');
+const {createCaseInCcd} = require("../helpers/ccdDataStoreApi");
+const {eventNames} = require('../pages/common/constants.js');
+const assert = require('assert');
+const {acceptCaseEvent, caseDetailsEvent} = require("../helpers/caseHelper");
+let caseNumber;
+
+const verifyState = (eventResponse, state) => {
```
Comment (dharrisMOJ) [inline src/test/end-to-end/paths/moveAcceptedCaseToCaseCloseStateErrorCheck.js:8]: I don't believe this method is used?

File: src/test/end-to-end/paths/moveRejectCaseToCaseCloseStateErrorCheck.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const testConfig = require('./../../config');
+const {createCaseInCcd} = require("../helpers/ccdDataStoreApi");
+const {eventNames} = require('../pages/common/constants.js');
+const assert = require('assert');
+const {rejectCaseEvent, caseDetailsEvent} = require("../helpers/caseHelper");
+let caseNumber;
+
+const verifyState = (eventResponse, state) => {
```
Comment (dharrisMOJ) [inline src/test/end-to-end/paths/moveRejectCaseToCaseCloseStateErrorCheck.js:8]: Similar to the previous test, i don't believe the verifyState is called?

File: src/test/end-to-end/pages/amendCaseDetails/amendCaseDetailsWithCasePosition.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+    const I = this;
+    await I.selectOption('#clerkResponsible', clerkResponcible);
+    await I.selectOption('#positionType', casePosition);
+    await I.selectOption('#fileLocation', physicalLocation);
+    await I.selectOption('#conciliationTrack', conciliationTrack);
+    await I.navByClick(commonConfig.continue);
+    await I.navByClick(commonConfig.continue);
+    await I.click(commonConfig.submit);
+    await I.see(commonConfig.moveAcceptedCaseToCloseCaseErrorCheck);
```
Comment (dharrisMOJ) [inline src/test/end-to-end/pages/amendCaseDetails/amendCaseDetailsWithCasePosition.js:14]: Should this be in commonConfig or Constants.js

File: src/test/end-to-end/paths/moveAcceptedCaseToCaseCloseStateErrorCheck.js (hunk 1 above)
Comment (phani-perla1) [reply src/test/end-to-end/paths/moveAcceptedCaseToCaseCloseStateErrorCheck.js:8]: fixed

File: src/test/end-to-end/paths/moveRejectCaseToCaseCloseStateErrorCheck.js (hunk 2 above)
Comment (phani-perla1) [reply src/test/end-to-end/paths/moveRejectCaseToCaseCloseStateErrorCheck.js:8]: fixed

File: src/test/end-to-end/pages/amendCaseDetails/amendCaseDetailsWithCasePosition.js (hunk 3 above)
Comment (phani-perla1) [reply src/test/end-to-end/pages/amendCaseDetails/amendCaseDetailsWithCasePosition.js:14]: Will take care that code improvements in next pr

**Review by dharrisMOJ — APPROVED** [review]
👍

**Pull request merged**